guides/source/command_line.md

```diff
@@ -443,24 +438,24 @@ Database adapter          sqlite3
 Database schema version   20180205173523
 ```
 
-### `assets`
+### `rails assets`
```

Comment:
Let's make this `rails assets:*`, and same for `db` and `tmp`, just because they're namespaces _only_, with no top-level command.